OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -430,13 +430,18 @@ public interface IObjectivesPanel
 	[RequireExplicitImplementation]
 	public interface INotifyObjectivesUpdated
 	{
-		void OnPlayerWon(Player winner);
-		void OnPlayerLost(Player loser);
 		void OnObjectiveAdded(Player player, int objectiveID);
 		void OnObjectiveCompleted(Player player, int objectiveID);
 		void OnObjectiveFailed(Player player, int objectiveID);
 	}
 
+	[RequireExplicitImplementation]
+	public interface INotifyWinstateChanged
```

Comment:
In my opinion, Win*S*tate with capital S would look better and also be more consistent with the enum.

Reply:
Done.